src/app/shared/case-editor/case-edit-page.component.ts

```diff
@@ -72,7 +73,8 @@ export class CaseEditPageComponent implements OnInit, AfterViewChecked {
       .filter(caseField => !this.caseFieldService.isReadOnly(caseField))
       .every(caseField => {
         let theControl = this.editForm.controls['data'].get(caseField.id);
-        return this.checkDocumentField(caseField, theControl) && this.checkOptionalField(caseField, theControl);
+        return this.isHidden(caseField, this.editForm.getRawValue())
```

Comment:
maybe it's a bit simpler to read if we move this isHidden check as another filter after 
```.filter(caseField => !this.caseFieldService.isReadOnly(caseField))```

Reply:
yes, absolutely better. Thanks for pointing out